### ISSUE TITLE
train.py出错

### ISSUE THREAD

**Issue by zhaoxiyubb** [LeslieZhoa/tensorflow-YOLO1#8]
OutOfRangeError (see above for traceback): RandomShuffleQueue '_2_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 16, current size 1)
	 [[Node: shuffle_batch = QueueDequeueManyV2[component_types=[DT_FLOAT, DT_DOUBLE], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](shuffle_batch/random_shuffle_queue, shuffle_batch/n)]]

**Comment by LeslieZhoa** [comment]
Hi @zhaoxiyubb  您是尝试用cpu训练吗？或许您可以尝试将batch_size调小，这应该是在tf.train.shuffle_batch出现的问题，您可以参考这两个问题的回答：[the link1](https://github.com/tensorflow/tensorflow/issues/7525), [the link2](https://stackoverflow.com/questions/42224327/outofrangeerror-see-above-for-traceback-fifoqueue-1-batch-fifo-queue-is-cl?noredirect=1)
希望对你有所帮助！

**Comment by lvsling** [comment]
我也出现了这个错误，不过我是用自己制作的训练集报这个错误，但是用下载下来的voc数据可以正常训练，训练模型对数据集有什么要求吗？我做过图像归一化，也resize过，但是都不行

**Comment by LeslieZhoa** [comment]
Hi @lvsling 请问你自己制作的数据集的label是做成voc数据的格式了吗？

**Comment by KyleWang-Hunter** [comment]
windows上还是没有跑起来
